index.js

```diff
@@ -176,67 +176,72 @@ module.exports = class BlindPeering {
 
     for (const mirrorKey of mirrors) {
       const ref = this._getBlindPeer(mirrorKey)
-      const cores = new Set()
 
       base.core.on('migrate', () => {
-        this._mirrorBaseBackground(ref, base, cores, all)
+        this._mirrorBaseBackground(ref, base, all)
       })
 
       base.on('writer', (writer) => {
-        if (!isStaticCore(writer.core) && !all) return
-        this._mirrorBaseWriterBackground(ref, base, writer.core)
+        const always = isStaticCore(writer.core) || all
+        this._mirrorBaseWriterBackground(ref, base, writer, always)
```

Comment:
This needs to be `writer.core` like before (or `_mirrorBaseWriterBackground` needs to expect a `writer` object instead of a `core` object)